packages/client/README.md

```diff
@@ -0,0 +1,3 @@
+# Slice Machine Client
```

Comment:
There's gotta be a tool to create reaedme's from js-doc style comments

Reply:
Maybe 🤷‍♂️ , I took it from the Core and wrote a small thing.